moped-database/views/project_list_view.sql

```diff
@@ -284,11 +284,11 @@ LEFT JOIN min_estimated_phase_dates mepd ON mp.project_id = mepd.project_id
 LEFT JOIN project_component_work_types pcwt ON mp.project_id = pcwt.project_id
 LEFT JOIN LATERAL (
     SELECT
-        mpn.project_note,
-        mpn.created_at AS date_created
-    FROM moped_proj_notes mpn
-    WHERE mpn.project_id = mp.project_id AND mpn.project_note_type = 2 AND mpn.is_deleted = false
-    ORDER BY mpn.created_at DESC
+        combined_project_notes.project_note,
+        combined_project_notes.created_at AS date_created
+    FROM combined_project_notes
+    WHERE (combined_project_notes.project_id = mp.project_id OR mp.should_sync_ecapris_statuses = true AND mp.ecapris_subproject_id IS NOT null AND combined_project_notes.ecapris_subproject_id = mp.ecapris_subproject_id) AND (combined_project_notes.note_type_name = any(ARRAY['Status Update'::text, 'eCapris Status Update'::text])) AND combined_project_notes.is_deleted = false
+    ORDER BY combined_project_notes.created_at DESC
```

Comment:
seeing this bit

```sql
 (combined_project_notes.note_type_name = any(ARRAY['Status Update'::text, 'eCapris Status Update'::text]))
```

makes me wonder if it would be worth adding an `is_status_update` boolean column to your view. it might provide for a tiny performance improvement and push a bit more business logic into the view. but i think it's a minor consideration.

and fwiw the `is_deleted` filter here seems not needed because your view is already excluding deleted notes.

Reply:
I like this idea a lot. Now that we have the additional status update type, I agree that handling that upstream in the new view would be an improvement and make what we are checking in this project_list_view more obvious too. 👍 

I'll remove the redundant filtering too - thanks for catching that! 🙏